resources/default-config.kdl

```diff
@@ -472,6 +472,9 @@ binds {
     Mod+Ctrl+U         { move-column-to-workspace-down; }
     Mod+Ctrl+I         { move-column-to-workspace-up; }
 
+    // Switches focus between the current and the previous workspace.
+    Mod+Tab            { focus-workspace-back-and-forth; }
```

Comment:
I think the original idea was not a new action, but a setting that makes it so pressing `focus-workspace N` while you're on N will send you to the previous workspace? This way there's no fallback behavior needed either.

Reply:
What do you think about this? I could change it myself to work like this if you'd like.

Reply:
I'm already working on an implementation. I'll also add `focus-workspace-no-auto` for the old behaviour.

Reply:
I think it should just be a setting, like `input { workspace-auto-back-and-forth; }` which will do that behavior if set. No need for a whole separate set of binds.

Reply:
Even though sway is capable of doing back and forth by using the keybind for the current workspace, back when I was using sway I preferred to use the separate keybind. Using the current workspace keybind requires the user to actually know what the number of the current workspace is.

Reply:
Hm. I thought the use-case here is that you know your, say, IRC is on workspace 1 and you go check it so you hit Mod+1 twice quickly (and don't need to remember your current workspace then).